kodilla-testing2/src/main/java/facebook/FacebookTestingApp.java

```diff
@@ -0,0 +1,47 @@
+package facebook;
+
+import config.WebDriverConfig;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.concurrent.TimeUnit;
+
+public class FacebookTestingApp {
+
+    public static final String XPATH_ACCEPT_ALL_BUTTON = "//div[contains(@class, \"_9fiw\")]/button[2]";
```

Comment:
Te klasy (np. `_9fiw`) wyglądają na coś ulotnego, co może się łatwo zmienić. Podobnie `button[2]` - polegasz na kolejności elementów na stronie, jak Facebook zamieni przyciski miejscami to już przestanie to działać. Proszę poszukaj lepszego punktu zaczepienia w drzewie DOM (najlepiej id elementu, możesz odnieść się też do tekstu przycisku, np. znaleźć button z napisem Accept All).